docs/funding/czi-grant-completed.md

```diff
@@ -0,0 +1,21 @@
+# Jupyter Accessibility Roadmap - Completed Items
+
+Created on: April 1, 2022
+
+Last updated on: April 1, 2022
+
+## Action items completed
+
+- Update the [JupyterLab accessibility project board](https://github.com/orgs/jupyterlab/projects/1)
+
+## In the works
+
+For items not yet completed, but next on the list, refer to the [Jupyter Accessibility Roadmap](czi-grant-roadmap.md)
+
+For the full scope of the two-year grant, all key items are listed on the [jupyter-a11y-mgmt repository's planning issues](https://github.com/orgs/Quansight-Labs/projects/5/views/1). 
+
+This roadmap's content is derived from
+- [The original grant proposal](https://github.com/jupyter/accessibility/blob/master/grant-applications/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf)
```

Comment:
I think this link is dead

Reply:
Correct link: https://github.com/jupyter/accessibility/tree/master/docs/funding/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf